README.md

```diff
@@ -49,6 +50,35 @@ This addon sets `CYPRESS_baseUrl` to DDEV's primary URL in the `docker-compose.c
 
 To display the Cypress screen and browser output, you must configure a `DISPLAY` environment variable.
 
+#### macOS
+
+ddev Cypress Setup (Mac)
+
+```bash
+# Install XQuartz
+brew install xquartz --cask
+
+# Run XQuartz
+open -a Xquartz
+```
+
+In the XQuartz preferences, go to the “Security” tab and make sure the “Allow connections from network clients” checkbox is checked.
+
+Now __restart your Mac__.  XQuartz will not properly be set to listen for X11 connections until you do this.
+
+```bash
+# Run the below command
+xhost + 127.0.0.1
```

Comment:
When we were experimenting, this was the path that made the most sense to us and worked consistently:
```suggestion
export DISPLAY=:0
xhost +
```

Reply:
Your suggestion worked fine for me, @rfay.